core/server/logging/PrettyStream.js

```diff
@@ -40,7 +40,8 @@ var _ = require('lodash'),
     };
 
 
-function PrettyStream() {
+function PrettyStream(format) {
+    this.format = format;
```

Comment:
suggestion: i prefer `mode`. I saw this naming also in other pretty streamers.
mode short or long.